### PR TITLE
OG Image Tags

### PR DESCRIPTION
**Pull request by MouseAndKeyboard**
Added OG image tags.
Maybe it would be better if the og:title tags was applied to all pages.

### CODE REVIEW

**Comment by Drewbi** [comment]
@MouseAndKeyboard Could you check why the home page is not showing it's card?
Also i believe some of the tests are fixed in auth but it might be a little buried so dw about it.

**Comment by MouseAndKeyboard** [comment]
> @MouseAndKeyboard Could you check why the home page is not showing it's card?
> Also i believe some of the tests are fixed in auth but it might be a little buried so dw about it.
@Drewbi 

It's working for me. When I installed the "Open Graph Preview" extension for Firefox it wouldn't show the homepage at first, I had to navigate around a bit (go to the about page, then go back to ./ ) to have it show up.

image

Still haven't added og:title and og:description tags. That's a TODO

File: pages/about.tsx [hunk 1]
```diff
@@ -10,6 +10,15 @@ const About = () => (
         name='description'
         content='Coders for Causes is a not for profit organisation that empowers charities and other not for profit organisations by connecting them with university students to develop technical solutions. We are a student-run club based in Perth, Western Australia with a wide range of clients. Whether you are looking for technical advice or a long term project, get in touch with us for more information.'
       />
+      <meta
+        property='og:image'
+        content='https://og-social-cards.dankestkush.vercel.app/**.%2Fabout**.png?theme=dark&md=1&fontSize=125px&images=https%3A%2F%2Fwebsite.codersforcauses.org%2Flogo%2Fcfc_logo_white_full.svg'
+      />
+      <meta property='og:site_name' content='Coders for Causes' />
+
+      <meta property='og:title' content='Who are we?' />
```
Comment (Drewbi) [inline pages/about.tsx:19]: Prolly change to `About Us`
```suggestion
      <meta property='og:title' content='About Us' />
```

**Comment by Drewbi** [comment]
Might have to add twitter card functionality to get discord to cooperate

**Comment by Drewbi** [comment]
Is this meant to be targeting `feature/auth`? It's ok to keep it on master and we'll merge the `cypress` branch first

**Comment by MouseAndKeyboard** [comment]
Targeting auth was a mistake, that being said, I think there are some /feature/auth specific pages which I added the OG tags for.

**Comment by MouseAndKeyboard** [comment]
Once the back end is ready we can add the tags to the the dynamically routed event and project pages.

**Comment by JeremiahPinto** [comment]
Twitter doesn't seem to render the card properly
image

**Pull request merged**